gems/smithy/lib/smithy/model.rb

```diff
@@ -2,11 +2,12 @@
 
 require_relative 'model/flattener'
 require_relative 'model/rbs'
-require_relative 'model/shape'
 require_relative 'model/operation_parser'
```

Comment:
nit: if we are ordering these by alphabetical order `rbs` goes after `operation_parser`